modules/aws/key-pair/variables.tf

```diff
@@ -4,9 +4,8 @@ variable "key_name" {
   nullable    = false
 }
 
-variable "download" {
-  description = "Whether to download the key to the local machine after creation"
-  type        = bool
-  nullable    = false
-  default     = false
+variable "filename" {
```

Comment:
I have wondered why we made this optional - if you don't download it you're scuppered if you ever want to log onto the nodes.